src/displayBackend/common/ConnectorBase.cpp

```diff
@@ -218,7 +218,7 @@ void ConnectorBase::edidPutDetailedTiming(edid* edidBlock, int index,
 	pixelData->misc = DRM_EDID_PT_SEPARATE_SYNC;
 }
 
-void ConnectorBase::edidPutTimings(edid* edidBlock)
+void ConnectorBase::edidPutTimings(edid* edidBlock, uint32_t xres, uint32_t yres)
```

Comment:
Well, I can see what you want to achieve, but it contradicts with the commit message a bit. You are writing:

> The methods are not using the class members or methods
> must be declared as static, there are
> ...
> ConnectorBase::edidPutTimings

But `ConnectorBase::edidPutTimings` actually uses `mCfgWidth` and `mCfgHeight`. 

Yes, you are refactoring this method to be static, but this should be mentioned in the commit message.